nengo/tests/test_neurons.py

```diff
@@ -281,7 +281,7 @@ def test_dt_dependence(Simulator, nl_nodirect, plt, seed, rng):
     """Neurons should not wildly change with different dt."""
     with nengo.Network(seed=seed) as m:
         m.config[nengo.Ensemble].neuron_type = nl_nodirect()
-        u = nengo.Node(output=WhiteSignal(0.1, 5), size_out=2)
+        u = nengo.Node(output=WhiteSignal(0.1, high=8), size_out=2)
```

Comment:
So this one is still all zero. Basically, we need `high < 1. / period` to get a non-zero signal. Is this check worth putting in WhiteSignal itself?